test/functional/helpers.lua

```diff
@@ -44,6 +44,8 @@ module.nvim_argv = {
   '--cmd', module.nvim_set,
   '--cmd', 'unmap Y',
   '--cmd', 'unmap <C-L>',
+  '--cmd', 'iunmap <C-U>',
+  '--cmd', 'iunmap <C-W>',
```

Comment:
for readability we should probably start a `module.nvim_unmap` variable to hold this.

we may also run into the max arg limit: https://github.com/neovim/neovim/blob/3954537b9e243fb3df3672d47fe3d9529fa4e544/src/nvim/main.h#L7-L8

up to now we have avoided passing a `-u` script to do the setup, possibly because it changes the startup sequence a bit. guess we can wait until we run into the limit...